splash/qtutils.py

```diff
@@ -8,12 +8,13 @@
 import time
 
 from PyQt5.QtCore import (QAbstractEventDispatcher, QDateTime, QObject,
-                          QUrl, QVariant, QEvent, Qt, QByteArray)
+                          QUrl, QVariant, QEvent, Qt, QByteArray, QSize)
 from PyQt5.QtGui import QKeyEvent
 from PyQt5.QtWidgets import QApplication
 from PyQt5.QtNetwork import QNetworkAccessManager, QNetworkReply, QNetworkProxy
 from PyQt5.QtWebKit import QWebSettings
 from PyQt5.QtWebKitWidgets import QWebFrame
+from PyQt5.QtWebEngineWidgets import QWebEngineProfile
```

Comment:
one weird thing that happens to me is that splash crashes on this error ```python bin/splash 
Traceback (most recent call last):
  File "bin/splash", line 3, in <module>
    from splash.server import main
  File "/home/pawel/scrapinghub/splash/splash/server.py", line 11, in <module>
    from splash.qtutils import init_qt_app
  File "/home/pawel/scrapinghub/splash/splash/qtutils.py", line 17, in <module>
    from PyQt5.QtWebEngineWidgets import QWebEngineProfile
ImportError: /home/pawel/Qt5.9.1/5.9.1/gcc_64/lib/libQt5WebEngineCore.so.5: symbol _ZSt24__throw_out_of_range_fmtPKcz version Qt_5 not defined in file libQt5Qml.so.5 with link time reference
``` with above traceback, but when I move this import up to line 9, before import of QtCore it works fine. Not sure if this is something about my local dev environment (I use ubuntu 18.04 with all libraries installed manually), or something else, it works fine on master though

Reply:
Does it happen in Docker, or are you installing everything without it?

Reply:
That's weird. 

Installing everything was tricky, I got quite a lot of errors like that until making it work, but haven't seen this particular issue. PyQT bundles its own QT, it can cause some issues, I'm not sure.

Reply:
ah sorry my fault @kmike  I see now even on master it crashes later with: "Cannot mix incompatible Qt library (version 0x50905) with this library (version 0x50901)"

so I probably installed something wrong, need to fix that